### ISSUE TITLE
can't install via opm

### ISSUE THREAD

**Issue by sebamontini** [aptise/lua-resty-peter_sslers#1]
a brand new amazon linux instance, just installed openresty via the centos repo.
but when i try to install via OPM y get this error
```
$ sudo opm get aptise/peter_sslers-lua-resty
* Fetching aptise/peter_sslers-lua-resty
  Downloading https://opm.openresty.org/api/pkg/tarball/aptise/peter_sslers-lua-resty-0.1.1.opm.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 11344  100 11344    0     0  12665      0 --:--:-- --:--:-- --:--:-- 12660
Package lua-resty-http-0.10 already installed.
ERROR: openresty is required but is not available according to resty:
```

any ideas? i have successfully installed a few other packages:
```
$ opm list
pintsized/lua-resty-http                                     0.10
bungle/lua-resty-prettycjson                                 1.6
openresty/lua-resty-dns                                      0.17
```

**Comment by jvanasco** [comment]
can you share the `opm` version and anything else about the distro ?

something probably changed in the package management.

**Comment by sebamontini** [comment]
sure, the distro is an Amazon linux 2016.09
```
Linux ip-172-17-3-199 4.4.41-36.55.amzn1.x86_64 #1 SMP Wed Jan 18 01:03:26 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux

Amazon Linux AMI release 2016.09

openresty-1.11.2.2-8.el6.x86_64
openresty-opm-1.11.2.2-8.el6.noarch

```

**Comment by sebamontini** [comment]
fixed! i was missing the `openresty-resty-1.11.2.2-8.el6.noarch` package
